### PR TITLE
fix some clippy warnings

### PR DESCRIPTION
**Pull request by remilauzier**

### CODE REVIEW

File: src/event/manager/mgr_shell.rs [hunk 1]
```diff
@@ -364,7 +364,7 @@ impl ManagerState {
         // make mgr const, but merely pretend it is in the public API.
         mgr.read_only = true;
 
-        for item in mgr.state.pending.pop() {
+        if let Some(item) = mgr.state.pending.pop() {
```
[the remark this answers is not in the document]
Comment (dhardy) [reply src/event/manager/mgr_shell.rs:367]: Yes, this should be:
```
while let Some(item) = mgr.state.pending.pop()  {
```

File: src/lib.rs [hunk 2]
```diff
@@ -23,7 +23,13 @@
 //! -   [Examples](https://github.com/kas-gui/kas/tree/master/kas-wgpu/examples)
 //! -   [Discuss](https://github.com/kas-gui/kas/discussions)
 
-#![allow(clippy::or_fun_call, clippy::never_loop)]
+// Use ``never_loop`` until: https://github.com/rust-lang/rust-clippy/issues/7397 is fixed
+#![allow(
+    clippy::or_fun_call,
+    clippy::never_loop,
+    clippy::module_inception,
```
[the remark this answers is not in the document]
Comment (remilauzier) [reply src/lib.rs:30]: It was ``mod draw;``

File: src/event/manager/mgr_shell.rs (hunk 1 above)
[the remark this answers is not in the document]
Comment (remilauzier) [reply src/event/manager/mgr_shell.rs]: I correct it.

File: src/lib.rs (hunk 2 above)
[the remark this answers is not in the document]
Comment (dhardy) [reply src/lib.rs:30]: Oh, `kas::draw::draw`. It's a private module though so seems fine.

**Review by dhardy — APPROVED** [review]
Looks good other than not being `rustfmt` clean.

I may ask you to rebase after #210.

**Pull request merged**